code/modules/projectiles/projectile/beams.dm

```diff
@@ -59,6 +59,9 @@
 	eyeblur = 0
 	impact_effect_type = /obj/effect/overlay/temp/impact_effect/blue_laser
 
+/obj/item/projectile/beam/disabler/laser_tag
+	damage = 0.0000000000000000000000000000000000000000000000000000000001

```

Comment:
yeah this is probably lower than whatever epsilon byond uses, almost certainly 0.